README.md

```diff
@@ -1,2 +1,171 @@
-# clinic-database
-This project aims to create database tables from a diagram
+# clinic Database
+
+## Getting Started
+
+This repository includes files with plain SQL that can be used to recreate a database:
+
+- Use [schema_based_on_diagram.sql](./schema_based_on_diagram.sql) to create all tables.
+
```

Comment:
- [ ] @Team It would have been nice if you combined this getting started section with the one below, to only have one getting started section, this will make your readme look more professional.